include/bitcoin/bitcoin/settings.hpp

```diff
@@ -88,6 +97,15 @@ class BC_API settings
 
     // This cannot be reactivated in a future branch due to window expiration.
     config::checkpoint bip9_bit1_active_checkpoint;
+
+    // Currency unit settings (uint64_t).
+    //--------------------------------------------------------------------------
+
+    uint64_t satoshi_per_bitcoin;
+    uint64_t initial_block_subsidy_bitcoin;
+    uint64_t recursive_money;
+    uint64_t retarget_subsidy_interval;
+    uint64_t no_retarget_subsidy_interval;
```

Comment:
`retarget_subsidy_interval` and `no_retarget_subsidy_interval` should be combined into `subsidy_interval`. This will eliminate need to pass both the `fork.retarget' setting and this value in cases where the interval is the only condition, which should simplify a bunch of code.

Reply:
:+1: